custom_components/reflex_clerk_api/base.py

```
@@ -1,7 +1,163 @@
+CLERK_REACT_LIBRARY = DEFAULT_CLERK_FRONTEND_VERSIONS.react_library
+CLERK_UI_LIBRARY = DEFAULT_CLERK_FRONTEND_VERSIONS.ui_library
+
+_clerk_frontend_versions = DEFAULT_CLERK_FRONTEND_VERSIONS
+
+
 class ClerkBase(rx.Component):
     # The React library to wrap.
     # `Show` is exported from `@clerk/react` (v6+), not `@clerk/clerk-react`.
-    library = "@clerk/react@^6.2.0"
+    library = CLERK_REACT_LIBRARY
+    lib_dependencies = list(DEFAULT_CLERK_FRONTEND_VERSIONS.dependency_libraries)
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)